core/src/main/java/hudson/PluginManager.java

```diff
@@ -1437,7 +1437,7 @@ public HttpResponse doPluginsSearch(@QueryParameter String query, @QueryParamete
                         jsonObject.put("releaseTimestamp", releaseTimestamp);
                     }
                     if (hasLatestVersionNewerThanOffered(plugin)) {
-                        jsonObject.put("newerVersionAvailableNotOffered", Messages.PluginManager_newerVersionExists(plugin.latest));
+                        jsonObject.put("newerVersionAvailableNotOffered", Messages.PluginManager_newerVersionExists(plugin.latest, plugin.wiki));
```

Comment:
Something's wrong with the merges in this PR, this should not show up in the diff.

Reply:
Good find! Should be sorted now :)